docs/source/devguide/dev_assisted_package_creation.rst

```diff
@@ -0,0 +1,104 @@
+    | # # #
+    | # # # This program is free software: you can redistribute it and/or modify it under
+    | # # # the terms of the NRLMMD License included with this program. This program is
+    | # # # distributed WITHOUT ANY WARRANTY; without even the implied warranty of
+    | # # # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the included license
+    | # # # for more details. If you did not receive the license, for more information see:
+    | # # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
+
+Developer Assisted Package Creation
+===================================
+
+Why We Assist With Package Creation
+-----------------------------------
+
+The GeoIPS Development team is committed to integrating a variety of geospatial
+processing routines into GeoIPS, or as a standalone packages that can interact with
+GeoIPS. Specifically we developers at CIRA are tasked with integrating proprietary
+software developed by researchers to work alongside GeoIPS. If you have access, see the
+`GeoIPS GLM Package <https://bear.cira.colostate.edu/overcast/geoips_glm>`_ for an
+example of a standalone package that implements GREMLIN and GLM
+(Global Lightning Mapper) routines working alongside GeoIPS.
+
+The purpose of integrating software like this is to elevate GeoIPS as a package that can
+process nearly any type of geospatial data, and output it in a variety of formats. For
+each standalone package that we create, we only elevate the capabilities of GeoIPS as a
+whole. Eventually, we'd like to see GeoIPS be the only package you need to handle your
+geospatial data routines.
+
+Use Cases That We'll Support
+----------------------------
+
+If you are a user, developer, or researcher that has software related to geospatial data
+processing and are unable to integrate that functionality into GeoIPS, you've come to
+the right place. We first request that the data processing you've implemented doesn't
+already exist in GeoIPS, or any of its corresponding packages. For a full list of public
+repositories that NRLMMD-GEOIPS supports, see
+`GeoIPS Repositories <https://github.com/orgs/NRLMMD-GEOIPS/repositories>`_.
+
+If the data routines you've created don't match any of the repositories linked above,
+then we've likely found a use case worth creating a separate GeoIPS package for.
+Assuming you've found a developer able to help you to get your processing routines
+integrated alongside GeoIPS, here's what we'll need from you.
+
+Information Needed for Developers
+---------------------------------
+
+In order to create a standalone GeoIPS Package implementing your routines, we'll need
+the following information to produce your expected output. This output could take on a
+variety of formats, but for this documentation, we'll assume it's some type of imagery,
+GeoTIFF, or NetCDF / HDF5 file.
+
+#. ``Data``
+
+   a. We'll take on any and all data that is needed to produce your expected output. The
```

Comment:
any and all? we don't have _any_ restrictions on the data we can intake?

Reply:
eg. can we handle foreign data, data from iran, etc. etc.

or trade secret IP ?

or 1000s of TBs?

Reply:
You're right, I updated this one. Good catch.